tests: allow to pass custom elasticsearch hosts to tests

ael-code opened the pull request:
make tests get the elasticsearch hosts from the LIBREANT_ES_HOSTS
 environment variable

ael-code committed:
tests: allow to pass custom elasticsearch hosts to tests

make tests get the elasticsearch hosts from the LIBREANT_ES_HOSTS
 environment variable

boyska commented:
haven't had the time to test it, but... is docker (that is, running elasticsearch - and only it) the real motivation for this?

ael-code committed:
fix travis-ci testing

new elasticsearch-5 package of travis container use x-pack plugin by
default. This introduced the necessity to authenticate all the requests
to the DB.

We need to use the LIBREANT_ES_HOSTS env variable to inform the test
suite about those credentials

ael-code commented:
now we have a real motivation... look at the second commit

ael-code commented:
@leophys the second commit fixes the travis building for ES 5.x
feel free to merge it
